packages/esm-patient-medications-app/src/components/medications-details-table.component.tsx

```diff
@@ -128,26 +129,29 @@ const MedicationsDetailsTable: React.FC<ActiveMedicationsProps> = ({
             </p>
           </div>
           <p className={styles.bodyLong01}>
-            {medication.orderReasonNonCoded ? (
+            {medication.orderReasonNonCoded && (
               <span>
                 <span className={styles.label01}>{t('indication', 'Indication').toUpperCase()}</span>{' '}
                 {medication.orderReasonNonCoded}
               </span>
-            ) : null}
-            {medication.quantity ? (
+            )}
+            {medication.quantity && (
               <span>
                 <span className={styles.label01}> &mdash; {t('quantity', 'Quantity').toUpperCase()}</span>{' '}
                 {medication.quantity} {medication?.quantityUnits?.display}
               </span>
-            ) : null}
-            {medication.dateStopped ? (
+            )}
+            {(medication.dateStopped || medication.autoStopDate) && (
               <span>
                 <span className={styles.label01}>
-                  &mdash; {t('discontinuedDate', 'Discontinued date').toUpperCase()}
+                  &mdash; {t('discontinuedDate', 'Discontinued date').toUpperCase()}:{' '}
+                  {formatDate(new Date(medication.dateStopped || medication.autoStopDate))}
                 </span>
-                {formatDate(new Date(medication.dateStopped))}
+                <Tag type="gray" className={styles.tag}>
+                  {t('discontinued', 'Discontinued').toUpperCase()}
+                </Tag>
```

Comment:
@ibacher should this tag also contain the discontinued date?

Reply:
Maybe as a tooltip? I think the goal here was just to better flag that these requests are discontinued. PS, I'd probably leave out the `toUpperCase()` and either change the translation here to `t('discontinued_Caps', 'DISCONTINUED')` or just drop it altogether (my preference).